src/components/SidePanel/index.js

```diff
@@ -69,64 +78,17 @@ SidePanel.propTypes = {
   closesOnOverlayClick: PropTypes.bool,
   footer: PropTypes.node,
   isOpen: PropTypes.bool,
-  animateOnMount: PropTypes.bool,
   onClose: PropTypes.func,
   title: PropTypes.node,
   header: PropTypes.node,
+  maxWidth: PropTypes.string,
```

Comment:
Leftover?

Reply:
it's passed to `StyledSidePanel`. Changed it to pass it explicitly.